docs/pages/admin-guides/access-controls/getting-started.mdx

```diff
@@ -1,253 +1,396 @@
-```
-</TabItem>
-<TabItem scope={["enterprise", "cloud"]} label="Commercial">
-```code
-# Once Alice logs back in, she will be able to view audit logs
-$ tctl users update alice --set-roles=editor,reviewer,auditor
-```
-</TabItem>
+In this step, you will create a local Teleport user with the `local-dev-access`
+role, then list available servers and connect the one with the label
+`env:local-dev`.
+
+1. Create a local user named `alice` with the `local-dev-access` role:
+
+   ```code
+   $ tctl users add alice --roles=local-dev-access
+   ```
+
+1. Follow the instructions in your terminal to sign in as `alice`.
+
+1. In your terminal, log out of your cluster and log in again:
+
+   ```code
+   $ tsh logout
+   $ tsh login --user=alice --proxy=<Var name="example.teleport.sh" />
+   ```
+
+1. List all servers available for your user to access. You should only see one:
+
+   ```code
+   $ tsh ls
+   Node Name    Address    Labels
+   ------------ ---------- -------------
+   ba2290caf694 ⟵ Tunnel   env=local-dev
+   ```
 
-</Tabs>
+   Since `alice` is denied access to servers with the `env:local-prod` label,
+   only the server with the `env:local-dev` label is available to connect to.
 
-Because Alice has two or more roles, permissions from those roles create a union. She
-will be able to act as a system administrator and auditor at the same time.
+1. Access the server using the value of the `Node Name` field as shown in `tsh
+   ls`:
 
-## Step 2/3. Map SSO users to roles
+   ```code
+   $ tsh ssh root@<Var name="ba2290caf694" />
+   ```
 
-Next, follow the instructions to set up an authentication connector that maps
-users within your SSO solution to Teleport roles.
+## Next steps
 
-### Teleport Enterprise
+In this guide, you created a Teleport role that allowed and denied access to SSH
+servers based on the labels that the servers were enrolled with. Read about more
+things you can do with Teleport roles.
 
-Create a SAML or OIDC application that Teleport can integrate with, then
-create an authentication connector that maps users within your application to
-Teleport roles.
+### Refine resource labels
 
-<Tabs>
-<TabItem label="SAML">
+Resource labeling is a crucial component of effective RBAC implementation in Teleport. Labels help you define which specific 
+resources users can access and why they need that access. Some common labels to consider using:
 
-Follow our [SAML Okta Guide](./sso/okta.mdx) to
-create a SAML application.
+- Environment (`dev`, `staging`, `production`)
+- Team ownership (`backend`, `frontend`, `data`)
+- Project or application (`auth-service`, `billing-api`)
+- Cost center or business unit (`marketing`, `engineering`)
 
-Save the file below as `okta.yaml` and update the `acs` field.
-Any member in Okta group `okta-admin` will assume a built-in role `admin`.
+Example label structure:
 
 ```yaml
-kind: saml
-version: v2
+kind: role
+version: v7
 metadata:
-  name: okta
+  name: senior-developer
```

Comment:
```suggestion
  name: backend-developer
```

In most organizations, access is based on what team you are part of, not your job level.